assets/js/blocks/collection-filters/inner-blocks/rating-filter/edit.tsx

```diff
@@ -0,0 +1,169 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import classnames from 'classnames';
+import { useBlockProps, InspectorControls } from '@wordpress/block-editor';
+import type { BlockEditProps } from '@wordpress/blocks';
+import {
+	Disabled,
+	Notice,
+	PanelBody,
+	ToggleControl,
+	withSpokenMessages,
+	// eslint-disable-next-line @wordpress/no-unsafe-wp-apis
+	__experimentalToggleGroupControl as ToggleGroupControl,
+	// eslint-disable-next-line @wordpress/no-unsafe-wp-apis
+	__experimentalToggleGroupControlOption as ToggleGroupControlOption,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import Block from './block';
+import { Attributes } from './types';
+import './editor.scss';
+
+const noRatingsNotice = (
+	<Notice status="warning" isDismissible={ false }>
+		<p>
+			{ __(
+				"Your store doesn't have any products with ratings yet. This filter option will display when a product receives a review.",
+				'woo-gutenberg-products-block'
+			) }
+		</p>
+	</Notice>
+);
+
+const Edit = ( {
+	attributes,
+	setAttributes,
+}: BlockEditProps< Attributes > ) => {
+	const {
+		className,
+		displayStyle,
+		showCounts,
+		showFilterButton,
+		selectType,
+	} = attributes;
+
+	const blockProps = useBlockProps( {
+		className: classnames( 'wc-block-rating-filter', className ),
+	} );
+
+	const getInspectorControls = () => {
```

Comment:
Another comment about convention here. Should we follow our new convention about putting the inspector in `components/inspector.tsx`?